### ISSUE TITLE
NS_BINDING_ABORTED error in every page

### ISSUE THREAD

**Issue by mirh** [EFForg/privacybadger#1368]
I was exploring (FF54b5) add-on debug console after reading #1318, when I saw it:
```
NS_BINDING_ABORTED: Component returned failure code: 0x804b0002 (NS_BINDING_ABORTED) [nsIStreamListener.onDataAvailable]
WebRequest.jsm:353

**Comment by ghostwords** [comment]
Does this actually break anything? This looks like a ["won't fix" Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341039).

**Comment by mirh** [comment]
Well, let's say that I have this feeling pb is slowing down the browser, but if you say so, I'm fine.

**Comment by ghostwords** [comment]
You're not wrong! We have lots of work to do on speeding up Privacy Badger, but it doesn't look like this particular error tells us anything.

**Comment by ghostwords** [comment]
Opened #1371 for general performance investigation.

**Comment by mirh** [comment]
Cool! Thank you.

**Comment by atomicwrites** [comment]
Hi, I know this is an old issue but I started having this problem a week or two ago where browsing has not been very responsive but more than that which is not really quantifiable, pages will randomly hang every once in a while and sometimes never load, and every time I check the network tab it has requests that say `NS_BINDING_ABORTED`. Sometimes reloading with "disable cache" turned on helps, but not always. After reading this I have tried disabling Privacy Badger for the site and the issue goes away. I know this is a pretty vague bug report but it looks like there is *some* kind of issue.
